modules/installing-odh-components.adoc

```diff
@@ -20,7 +20,7 @@ endif::[]
 | Component | Required Operators | Catalog
 
 | ksersve
```

Comment:
Typo? Should this be kserve?

Reply:
Yes! I did see it.
But I approved anyway, since it is old. It is up to you and @fabiobrz if you want the typo solved here.

Reply:
I've approved, but suggest fixing this to make sure the user is clear on the component name.